canopen/nmt.py

```diff
@@ -73,6 +73,9 @@ def send_command(self, code: int):
 
     @property
     def state(self) -> str:
+        return self.get_state()
+
+    def get_state(self) -> str:
```

Comment:
If we want to go this route, this adds another layer of indirection, which has a small runtime penalty.  Couldn't the properties be created by specifying the `get_*()` and `set_*()` methods directly as `fget` and `fset` (if that's what they're called...) parameters?  If the decorated method is just a wrapper itself, there is no real advantage in using a decorator anymore.